accounts.md

```diff
@@ -23,21 +23,28 @@ POST `https://service.centrapay.com/api/accounts`
 curl -X POST "https://service.centrapay.com/api/accounts" \
 -H "x-api-key: 1234" \
 -H "content-type: application/json" \
--d '{"name":"Centrapay Cafe"}'
+-d '{"name":"Centrapay Cafe", "type": "org"}'
 ```
 
 **Required Fields**
 
-| Field |  Type  |       Description       |
-| ----- | ------ | ----------------------- |
-| name  | String | The name of the account |
+| Field |  Type  |                    Description                     |
+| ----- | ------ | -------------------------------------------------- |
+| name  | String | The name of the account                            |
+| type  | String | Account type, must be either 'org' or 'individual' |
 
 **Example response payload**
 
 ```
 {
   "id": "Jaim1Cu1Q55uooxSens6yk",
-  "name": "Centrapay Cafe"
+  "name": "Centrapay Cafe",
+  "type": "org",
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "createdAt": "2020-06-12T01:17:46.499Z",
+  "modifiedAt": "2020-06-12T01:17:46.499Z",
+  "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "version": 1
```

Comment:
I am thinking we should make all integers strings when serialized to JSON (ie versions and currency values). Internally we should switch to using BigInt and implement BigInt.toJSON(): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Use_within_JSON